packages/webpack-plugin/src/plugin.ts

```diff
@@ -3,22 +3,20 @@ import { theme } from "@kuma-ui/sheet";
 import path from "path";
 import { readdirSync } from "fs";
 import { getUserTheme } from "./getUserTheme";
+import { createRequire } from "module";
 
-// type WebpackPluginOption = {};
+declare const require: NodeRequire;
+declare const __ESM__: boolean;
+const _require = __ESM__ ? createRequire(import.meta.url) : require;
```

Comment:
@taishinaritomi 
The origin of the `__ESM__` variable are not immediately clear from this context alone. It would be helpful to include a comment explaining that this variable is declared in ``sup.config.ts ` and its intended use, for clarity to anyone reviewing or modifying this code in the future 🙏.